go/test/endtoend/cluster/cluster_process.go

```diff
@@ -199,7 +209,7 @@ func (cluster *LocalProcessCluster) StartKeyspace(keyspace Keyspace, shardNames
 // StartVtgate starts vtgate
 func (cluster *LocalProcessCluster) StartVtgate() (err error) {
 	vtgateHTTPPort := cluster.GetAndReservePort()
-	vtgateGrpcPort := cluster.GetAndReservePort()
+	vtgateGrpcPort := VtGateGrpcPort //cluster.GetAndReservePort()
```

Comment:
instead of using a hardcoded port number I suggest the following:
```
VtGateGrpcPort = cluster.GetAndReservePort()
```
Then use this to initialize VtgateProcessInstance (line 217)

Reply:
Done